.github/dependabot.yml

```diff
@@ -0,0 +1,35 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
```

Comment:
We can probably leave out bundler, terraform, npm, gomod and docker for this repo.

Reply:
Assuming the intention is for this to be a library. The other updates are only necessary if we intend to build docker images etc. for production. It wouldn't hurt too much to leave docker in while we have a docker compose file, though.

Reply:
I'll leave docker in for now - being able to use `docker compose up localstack` has been quite useful for checking the pipeline works. There's other stuff we can remove in another PR